### ISSUE TITLE
support inference endpoint on pertained model directly

### ISSUE THREAD

**Issue by jackie930** [aws-samples/amazon-sagemaker-mask-r-cnn-pytorch#1]
hi,

I have built an additional function that can build a SageMaker endpoint directly from pretrained-model, and serve on gpu/cpu instance. wanna to check can I submit a pr on this?

**Comment by billdoors** [comment]
that sounds great. Please do so. thanks.
